rules/windows/process_creation/proc_creation_win_lolbin_createdump.yml

```
@@ -5,9 +5,9 @@ description: Detects uses of the createdump.exe LOLOBIN utility to dump process
 references:
     - https://www.crowdstrike.com/blog/overwatch-exposes-aquatic-panda-in-possession-of-log-4-shell-exploit-tools/
     - https://twitter.com/bopin2020/status/1366400799199272960
-author: Florian Roth (Nextron Systems), Nasreddine Bencherchali
+author: Florian Roth (Nextron Systems), Nasreddine Bencherchali (Nextron Systems)
 date: 2022/01/04
-modified: 2022/08/19
+modified: 2023/02/04
```

no need to update for author change
```suggestion
modified: 2022/08/19
```